docs/resources/logbook/2023_lhc.md

```diff
@@ -10,28 +10,41 @@ The tables below can be sorted by clicking next to the column headers.
+[meas_in_ramp_sum]:                         https://be-op-logbook.web.cern.ch/elogbook-server/#/logbook?logbookId=322&dateFrom=2023-04-06T00%3A00%3A00&dateTo=2023-04-06T23%3A59%3A59&eventToHighlight=3744954 
 [a4_corrections_30cm]:                      https://be-op-logbook.web.cern.ch/elogbook-server/#/logbook?logbookId=1081&dateFrom=2023-04-08T00%3A00%3A00&dateTo=2023-04-08T23%3A59%3A59&eventToHighlight=3745378
 [a4_corrections_30cm_sum]:                  https://be-op-logbook.web.cern.ch/elogbook-server/#/logbook?logbookId=1081&dateFrom=2023-04-08T00%3A00%3A00&dateTo=2023-04-08T23%3A59%3A59&eventToHighlight=3745433
 [nl_studies_injection]:                     https://be-op-logbook.web.cern.ch/elogbook-server/#/logbook?logbookId=1081&dateFrom=2023-04-08T00%3A00%3A00&dateTo=2023-04-08T23%3A59%3A59&eventToHighlight=3745829
 [nl_studies_injection_sum]:                 https://be-op-logbook.web.cern.ch/elogbook-server/#/logbook?logbookId=322&dateFrom=2023-04-08T23%3A00%3A00&dateTo=2023-04-09T07%3A00%3A00&eventToHighlight=3746004
+[ions_virgin_coupl]:                        https://be-op-logbook.web.cern.ch/elogbook-server/#/logbook?logbookId=1081&dateFrom=2023-04-10T00%3A00%3A00&dateTo=2023-04-10T23%3A59%3A59&eventToHighlight=3746402
+[nl_studies_injection]:                     https://be-op-logbook.web.cern.ch/elogbook-server/#/logbook?logbookId=1081&dateFrom=2023-04-16T00%3A00%3A00&dateTo=2023-04-16T23%3A59%3A59&eventToHighlight=3751170
+[nl_studies_injection_sum]:                 https://be-op-logbook.web.cern.ch/elogbook-server/#/logbook?logbookId=1081&dateFrom=2023-04-16T00%3A00%3A00&dateTo=2023-04-16T23%3A59%3A59&eventToHighlight=3751258
 
 
 
 <!-- Tooltips -->
 *[Injection Linear Optics]:  Corrections with total phase-advance. DA comparison with MO and chroma (improved!). Injection optics with former (2021) corrections validated, fine corrections for normalized dispersion.
-*[Ramp and Squeeze Linear Optics]: Measurements and validation of optics from 200cm down to 30cm, including K-Mod. New corrections at 120cm. All others agree well with corrections from the 2023 MD last year. 
+*[Ramp and Squeeze Linear Optics]: Measurements and validation of optics from 200cm down to 30cm, including K-Mod. First new corrections at 120cm, applied from 200cm to 120cm. Bad IP8. All others agree well with corrections from the 2023 MD last year. 
+*[Measurements in the Ramp + Validation]: Measurements in the Ramp, Validation of 120cm optics with corrections done at 04-05 to lower IP8 beating. 
 *[a4 Corrections at 30cm]: Measurement of the f1012 and f1210 RDTs and corrections tests at 30cm. Some K-Mod measurement at 87cm.
 *[NL Studies at Injection]: Measurement of several RDTs with and without landau octupoles with both 2022 and 2023 optics.
+*[Ions Virgin + Local Coupling]: Get beams to end of ramp. Managed only after 2nd try (dump due to kick close to tune). Minor coupling correction.
```

Comment:
Blunders and stuggles to get to the end of the ramp are irrelevant. What matters for the tooltip is only what was observed and achieved in the shift.

Reply:
Ok